scripts/check_coverage.py

```
@@ -160,19 +163,114 @@ def is_executable_source_line(
+        "pub const fn ",
+        "pub(crate) fn ",
+        "pub(crate) const fn ",
+        "const fn ",
+        "fn ",
+    )):
         return False
     if text.startswith("pub struct ") or text.startswith("struct "):
         return False
     if text.startswith("pub enum ") or text.startswith("enum "):
         return False
     if text.startswith("Ok(Self") or text in {")}", "})", "})"}:
```

Ruff (0.16.1)

[warning] Sets should not contain duplicate item `"})"`

Remove duplicate item

(B033)